pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -8,3 +8,42 @@ Elements included in the accessibility tree of platform specific accessibility A
 The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are included in the accessibility tree can be found in the [HTML accessibility API mappings (working draft)](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings (working draft)](https://www.w3.org/TR/svg-aam/).
 
 > **Note:** Users of assistive technologies might still be able to interact with elements that are not included in the accessibility tree. An example of this is a [focusable](#focusable) element with an `aria-hidden` attribute with a value of `true`. Such an element could still be interacted with using sequential keyboard navigation regardless of the assistive technologies used, even though the element would not be included in the accessibility tree.
+
+#### Examples
+
+This `span` element is included in the accessibility tree (by default, elements are included in the accessibility tree).
+
+```html
+<span>ACT rules</span>
+```
+
+This `span` element is not included in the accessibility tree because it is hidden to everybody by the CSS property.
+
+```html
+<span style="display:none">ACT rules</span>
+```
+
+This `span` element is not included in the accessibility tree because it is explicitly removed by the `aria-hidden` attribute.
+
+```html
+<span aria-hidden="true">ACT rules</span>
+```
+
+This `span` element is positioned off screen, hence is not [visible](#visible), but is nonetheless included in the accessibility tree.
+
+```html
+<span style="position: absolute; top:-9999em">ACT rules</span>
+```
+
+Although the `span` element with an `id` of "label" is not itself included in the accessibility tree, it still provides an [accessible name](#accessible-name) to the other `span`, via the `aria-labelledby` attribute. Thus, it is still indirectly exposed to users of assistive technologies.
```

Comment:
This might be confusing to some readers. The element is not in the accessibility tree but is indirectly exposed to users of AT (thus it seems it is in accessibility tree). Perhaps we can leave the last sentence out?

Reply:
Or changed into something explicit like "Thus, it is still indirectly exposed to users of assistive technologies without being included in the accessibility tree"

Reply:
I'm more in line with @ShadowBB on that one. The goal of that sentence is to say that "just because you put a `aria-hidden` attribute, it does not mean you can do whatever you want without breaking accessibility".

Typically,
```html
<img src="https://picsum.photos/100" aria-hidden=true id="img" />
<button aria-labelledby="img">Foo</button>
```
Although the image is not in the accessibility tree, not giving it a name is still creating accessibility issues (namely, not giving a name to the button).

Reply:
@carlosapaduarte @ShadowBB I tried to improve the description of the example to better communicate the intend.